Libplanet.Net/Protocols/KBucket.cs

```diff
@@ -86,7 +86,7 @@ public void Clear()
         /// </summary>
         /// <param name="peer">The <see cref="BoundPeer"/> to remove.</param>
         /// <returns>
-        /// <c>true</c> if <paramref name="peer"/> was removed, <c>false</c> otherwise.
+        /// <see langword="true"/> if <paramref name="peer"/> was removed, <see langword="false"/> otherwise.
```

Comment:
Character per line limit is set to `100`. Otherwise it won't build. You'd have to change the lines manually for this. Please try building on your local machine to see if there are any offending lines. 😗

Reply:
sure